Src/IronPythonTest/Cases/IronPythonCasesManifest.ini

```diff
@@ -174,33 +174,18 @@ Ignore=true
 [IronPython.hosting.editor_svcs.test_tokencategorizer]
 Ignore=true
 
-[IronPython.interop.net.derivation.test_method_override]
-Ignore=true
-
 [IronPython.interop.net.derivation.test_property_override]
 IsolationLevel=PROCESS # causes a failure in IronPython.test_statics
 
 [IronPython.interop.net.field.test_field_misc]
 Ignore=true
 
-[IronPython.interop.net.field.test_fields_inside_enum]
-Ignore=true
-
-[IronPython.interop.net.field.test_instance_fields]
-Ignore=true
-
 [IronPython.interop.net.test_accessibility]
 Ignore=true
 
 [IronPython.interop.net.test_loadorder]
 Ignore=true
 
-[IronPython.interop.net.type.test___clrtype]
```

Comment:
I haven't investigated the specific causes of the failures during CI, but to resolve change to:
```ini
[IronPython.interop.net.type.test___clrtype]
IsolationLevel=PROCESS # causes a failure in IronPython.test_attrinjector

[IronPython.interop.net.type.test_reachtype]
IsolationLevel=PROCESS # causes failures in IronPython.test_ast and IronPython.scripts.test_cgcheck
```

Reply:
Thanks a lot! Updated.